pkg/couchdb/couchdb.go

```diff
@@ -19,6 +20,10 @@ import (
 	"github.com/labstack/echo"
 )
 
+// InfiniteString is the unicode character "\uFFFF", useful in query as
+// a upperbound for string.
+const InfiniteString = string(unicode.MaxRune)
```

Comment:
Is is defined as `uFFFF` here: https://github.com/cozy/cozy-stack/blob/master/pkg/couchdb/mango/query.go#L155. We should probably use the same name to be consistent